core/src/main/scala/org/apache/spark/sql/crossdata/models/EphemeralTableModel.scala

```diff
@@ -18,7 +18,7 @@ package org.apache.spark.sql.crossdata.models
 import org.apache.spark.sql.types.StructType
 
 case class EphemeralTableModel(name: String,
-                               schema: StructType,
+                               schema: Option[StructType],
```

Comment:
Change to the last position in argument list, then we can create EphemeralTableModel(name, options)